Sources/Vapor/Utilities/AnySendableHashable.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// A `Sendable` version of the standard library's `AnyHashable` type.
+public struct AnySendableHashable: @unchecked Sendable, Hashable, ExpressibleByStringLiteral {
+    // Note: @unchecked Sendable since there's no way to express that `wrappedValue` is Sendable, even though we ensure that it is in the init.
+    let wrappedValue: AnyHashable
+    
+    public init(_ wrappedValue: any Hashable & Sendable) {
```

Comment:
```suggestion
    public init(_ wrappedValue: some Hashable & Sendable) {
```
This brings it in line with `AnyHashable`'s initializer:
```swift
@frozen public struct AnyHashable {
    public init<H>(_ base: H) where H : Hashable
```
(which is functionally identical to `public init(_ base: some Hashable)`)